app/validators/base_validator.rb

```diff
@@ -199,4 +201,9 @@ def vat_exceeds_max?(vat:, net:)
     max_vat = VatRate.vat_amount(net, @record.claim.vat_date, calculate: true)
     vat.round(2) > max_vat.round(2)
   end
+
+  def looks_like_a_case_number?(attribute)
+    return if attr_blank?(attribute)
+    @record.__send__(attribute).match?(CASE_NUMBER_OR_URN_PATTERN)
```

Comment:
should this not be 
```
 @record.__send__(attribute).match?(CN_PATTERN)
```

Reply:
I don't think so. The CASE_NUMBER_OR_URN_PATTERN regex is a bit looser to try and pick up things that could be a case number. If looks_like_a_case_number? is true this then means the case number is validated against CN_PATTERN.